incubator/elasticsearch/templates/client-deployment.yaml

```diff
@@ -26,8 +26,10 @@ spec:
 {{- end }}
       securityContext:
         fsGroup: 1000
-      {{- if eq .Values.client.antiAffinity "hard" }}
+      {{- if or .Values.client.antiAffinity .Values.client.nodeAffinity }}
```

Comment:
this will basically always be true, since antiAffinity has a value "soft" if will be true

Reply:
I'm not sure to understand what you mean by saying "it should be opt-in by changing the values actively by the end-user of the chart at deployment time" ?

I've made nodeAntiAffinity opt-in already, and did not change the default for antiAffinity. I was just saying that a user may actively opt-out the existing antiAffinity and keep default for this new nodeAffinity, in this case the "affinity:" line should not appear.

Reply:
Are you ok with the anwser? Can I resolve conversation?